src/staff-administrator/models/staff-job-titles.ts

```diff
@@ -0,0 +1,84 @@
+export class StaffJobTitles {
+  public static jobTitles = [
```

Comment:
This stuff should not be hard coded into the app. Needs to be replaced with reference data calls ASAP